docs/build-process.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Installing Taskcluster
+order: 10
+---
+
+Taskcluster installation breaks down into two phases:
+
+ * Build a Taskcluster release
+ * Deploy that release
+
+# Build
+
+The build phase takes a [build spec](./build-spec) and source code and, along with lots of external resources like docker images, third-party packages, and so on, and produces a "release".
+
+The output of the build process is a [release](./release).
+This contains references to the built artifacts as well as a copy of the build-time configuration.
+
+# Deploy
+
+The deploy phase takes a [release](./release) and a [deploy spec](./deploy-spec) and deploys Taskcluster as described in the deploy spec.
```

Comment:
there is something to work out here... probably, the release is a deploy spec, and the thing you supply is `deploy-config`.

Reply:
That sounds like a naming difference.  I'm using "spec" to mean something that users write.  But we decided yesterday that we will all try different approaches here, so please do experiment :)